db/migrate/20210914114030_add_pseudomize_status_to_decidim_components.decidim.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+# This migration comes from decidim (originally 20210810081926)
+
+class AddPseudomizeStatusToDecidimComponents < ActiveRecord::Migration[5.2]
+  def change
```

Comment:
Hi, shouldn't we use up rather than change for adding a new column ?
And so shouldn't we add a new method down for removing the newly created column in case of rollback ?
```suggestion
  def up
```